### PR TITLE
build-fail-blacklist: Remove abyss

### PR DESCRIPTION
**Pull request by sjackman**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR does something else (explain below).

Remove `abyss` from `build-fail-blacklist`.

### CODE REVIEW

**Comment by sjackman** [comment]
Is this PR sufficient to trigger a CI build of `abyss`?

**Comment by npavlovikj** [comment]
That is enough @sjackman  - however, the download url is not existing, so the build fails with "Error: HTTP 404 NOT FOUND for url <http://www.bcgsc.ca/platform/bioinfo/software/abyss/releases/2.1.0/abyss-2.1.0.tar.gz>". @druvus did some fixes in #9855, but the recipe still needs some more tweaking to build properly.

**Comment by sjackman** [comment]
```
06:08:14 BIOCONDA INFO BUILD SUCCESS abyss-2.1.0-h09b0a5c_0.tar.bz2
…
[Jul 11 06:09:20] SERR ABYSS-P: error while loading shared libraries: libsqlite3.so.0: cannot open shared object file: No such file or directory
```
https://circleci.com/gh/bioconda/bioconda-recipes/17500

`sqlite` is not listed a dependency of the recipe, yet `configure` appears to have picked up sqlite opportunistically. I'll add an explicit `./configure --without-sqlite`.

**Comment by sjackman** [comment]
💚 and life is good.

File: recipes/abyss/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ build:
   number: 0
 
 source:
-  url: http://www.bcgsc.ca/platform/bioinfo/software/{{ name|lower }}/releases/{{ version }}/{{ name|lower }}-{{ version }}.tar.gz
```
Comment (bgruening) [inline recipes/abyss/meta.yaml]: @sjackman if you do not increase the build number (or the version number and reset the build number to 0) the pkg will not be build.

Comment (druvus) [reply recipes/abyss/meta.yaml]: Abyss version 2.1 were never previously built due to blacklisted so I think this is good to go.

**Comment by sjackman** [comment]
Thanks for merging, Andreas!

**Pull request merged**